src/lib/health-checks.js

```diff
@@ -1,13 +1,20 @@
 const checkFailing = require('./check-failing');
+const errorRateCheck = require('./error-rate-check');
 
 module.exports = (app, options, meta) => {
-	const healthChecks = options.healthChecks;
 	const defaultAppName = `Next FT.com ${meta.name} in ${process.env.REGION || 'unknown region'}`;
 
+	const defaultChecks = [];
+	if (!options.skipDefaultErrorRateCheck) {
+		defaultChecks.push(errorRateCheck(meta.name));
+	}
```

Comment:
maybe we shouldn't give the option to turn this off, just the option to set the threshold...